firstvoices/backend/models/sites.py

```diff
@@ -241,8 +241,6 @@ class SiteFeature(BaseSiteContentModel):
         max_length=DEFAULT_TITLE_LENGTH,
         blank=False,
         validators=[validate_slug],
-        db_index=True,
```

Comment:
We should add a validation so that the key is still unique for each site else we can end up with duplicate keys for one site.
This can be done using `unique_together`.
Reference: https://docs.djangoproject.com/en/5.0/ref/models/options/#unique-together

Reply:
The unique together condition is already in the model, in the Meta class below

Reply:
Oh yes, I missed that. My bad.